app/models/piece.rb

```diff
@@ -49,4 +49,15 @@ def is_obstructed?(col, row)     ## pass in rank and file of the square we want
     end
     false
   end
+
+  def move_to!(new_col, new_row)
+    current_col = self.file
+    current_row = self.rank
+    pieces = Piece.where(file: new_col, rank: new_row, game: game, is_captured: false)
+      captured_piece = pieces.first
```

Comment:
quick style change: lines 57 to 61 shouldn't be indented

Reply:
Thanks - I've changed that.